### ISSUE TITLE
Got error from request

### ISSUE THREAD

**Issue by 0xB455** [defaultnamehere/zzzzz#22]
Hello,

you should note python-dev and libssl-dev as requirement. I had some SSL issues which I got rid by installing both and going for

> pip install 'requests[security]'

afterwards. This got me going.

However now I get the following response. Not sure where the request is failing.

```
#python fetcher.py
>
Got error from request, restarting...
>
Got error from request, restarting...
>
Got error from request, restarting...
>
Got error from request, restarting...
^CTraceback (most recent call last):
  File "fetcher.py", line 177, in <module>
    time.sleep(SLEEP_TIME)
KeyboardInterrupt
```

I would debug the HTTP requests but it is all TLS1.2 encrypted. Any idea how to proceed?

**Comment by tdurieux** [comment]
Double Check the content of your `SECRETS.txt` file

**Comment by 0xB455** [comment]
Yup, just fixed it by providing the proper cookies... it was an issue with SECRETS.txt as also mentioned in [https://github.com/defaultnamehere/zzzzz/issues/23](https://github.com/defaultnamehere/zzzzz/issues/23)

**Comment by a9e78rauijasdf** [comment]
Im having the same problem. ClientID seems to change now and then. Userid is correct. Cookie is also provided as it should - no ' or " around the cookie data, which is copied directly via chrome dev tools (copy as curl (bash)). Any ideas?
What was the specific problem, TheBazZz?

**Comment by Karlheinzniebuhr** [comment]
Any solution to this? 
The error message should be clearer also

**Comment by ltrgoddard** [comment]
Make sure you're copying the cookie using "Copy as cURL (bash)" rather than "Copy as cURL (cmd)"---the latter adds double quotes around certain characters, e.g. %, which you don't want.

**Comment by a9e78rauijasdf** [comment]
Im afraid it does not solve the problem in my case. Perhaps a clear example of the txt file would provide help?

**Comment by garrymann** [comment]
I am also having the same issue. **uid** is obvious. **cl_id** is changing by sessions but can be found.
What is exactly required from **cookie** value? The **xs** portion? What if it contains a % char?

**Comment by ltrgoddard** [comment]
Copy everything from and including 'datr' up until the final quotation mark
(before '--compressed' or whatever).

**Comment by vicprox** [comment]
I get :
Got error from request, restarting...
>
from fetcher.py too, is this tool still working?

**Comment by shymega** [comment]
@vicprox from what i understand, the API has changed, so some investigation and fixes are required for `fetcher.py` to work again. See #49 for more information.
